autopush/crypto_key.py

```diff
@@ -0,0 +1,82 @@
+"""Crypto-Key header parser and manager"""
+
+
+class CryptoKeyException(Exception):
+    """Invalid CryptoKey"""
+
+
+class CryptoKey(object):
+    """Parse the Crypto-Key header per
+    http://tools.ietf.org/html/draft-ietf-httpbis-encryption-encoding-00#section-4
+
+    """
+    _values_ = []
```

Comment:
This is very unorthodox Python. Double underscores on both sides signify built-in override-able methods, I've never seen or heard of single underscores on both sides. Single leading underscore is generally a private var, and double leading underscore is a special private class var that gets Python class mangling applied. What's the rationale for this new approach?